tests/integration/standard/test_udts.py

```diff
@@ -599,6 +598,9 @@ def test_nonprimitive_datatypes(self):
                 if nonprim_datatype == "map":
                     type_string = "{0}_{1} {2}<{3}, {3}>".format(chr(start_index + i), chr(start_index + j),
                                                                  nonprim_datatype, datatype)
+                elif nonprim_datatype == "tuple":
+                    type_string = "{0}_{1} frozen<{2}<{3}>>".format(chr(start_index + i), chr(start_index + j),
+                                                            nonprim_datatype, datatype)
```

Comment:
FWIW, this test is currently running on the java-driver, and will be running on the ruby-driver. Even if it will be fixed in 2.1.3, explicitly having "frozen" is forwards-compatible for 2.1.3+